src/Microsoft.DotNet.Build.CloudTestTasks/AzureHelper.cs

```diff
@@ -0,0 +1,373 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AzureHelper.cs" company="">
+//   
+// </copyright>
+// <summary>
+//   The azure helper.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    #region
+
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Security.Cryptography;
+    using System.Text;
+    using System.Text.RegularExpressions;
+
+    #endregion
+
+    /// <summary>
+    /// The azure helper.
+    /// </summary>
+    public static class AzureHelper
+    {
+        /// <summary>
+        /// The storage api version.
+        /// </summary>
+        public static readonly string StorageApiVersion = "2015-04-05";
+
+        /// <summary>
+        /// The sas access type.
```

Comment:
Since it's in documentation here, "sas" should be one of "SAS", "Shared Access Signature (SAS)", or "SAS (Shared Access Signature)".